components/sidebar_right/sidebar_right.jsx

```diff
@@ -55,6 +76,27 @@ export default class SidebarRight extends React.PureComponent {
         }
     }
 
+    determineTransition = () => {
+        var transitionInfo = window.getComputedStyle(this.sidebarRight.current).getPropertyValue('transition');
+        var hasTransition = transitionInfo !== 'all 0s ease 0s';
```

Comment:
nit: Generally we try to use `const` or `let` as appropriate instead of `var`

Reply:
Whoops. Old habits die hard :P